src/fluree/db/util/core.cljc

```diff
@@ -426,3 +426,12 @@
   (-> json-ld
       (get-first k)
       :id))
+
+(defn parse-opts
```

Comment:
I'm not sure I love it being in `util.core`, but I think it works. As we do the new transaction semantics I think this sort of thing will be consolidated more formally anyways.

Reply:
Do you see a better place for it? I didn't spot a better place for something shared by both queries and transactions.